=== modified file 'clc/modules/postgresql/conf/scripts/setup_db.groovy'
--- clc/modules/postgresql/conf/scripts/setup_db.groovy	2012-01-13 20:11:37 +0000
+++ clc/modules/postgresql/conf/scripts/setup_db.groovy	2012-01-17 18:35:37 +0000
@@ -94,13 +94,13 @@
      // Static definitions of postgres commands and options
  
      private static String EUCA_DB_DIR  = SubDirectory.DB.getFile().getPath() + "/data/";
-     private static String PG_BIN = "/usr/pgsql-9.1/bin/pg_ctl";
+     private static String PG_BIN = "/usr/bin/pg_ctl";
      private static String PG_START = "start";
      private static String PG_STOP_OPTS = "-mf"
      private static String PG_STOP = "stop";
      private static String PG_PORT_OPTS2 = "-o -h0.0.0.0/0 -p8777 -i";
      private static String PG_DB_OPT = "-D";
-     private static String PG_INITDB = "/usr/pgsql-9.1/bin/initdb ";
+     private static String PG_INITDB = "/usr/bin/initdb ";
      private static String PG_X_OPT = " -X ";
      private static String PG_X_DIR =  SubDirectory.TX.getFile().getPath();
      private static String PG_USER_OPT = " -Ueucalyptus ";

